test/Test/Db.hs

```diff
@@ -1,2 +1,64 @@
-module Test.Db where
-    
\ No newline at end of file
+module Test.Db 
+       ( dbSpecs
+       ) where
+
+import IW.App (AppEnv, App)
+import IW.Db (getIssues, upsertIssues)
+
+import Test.Assert (equals, succeeds)
+import Test.Data (invalidIssue, updateIssue, validIssue)
+import Test.Hspec (Spec, describe, it, pending)
+
+
+dbSpecs :: AppEnv -> Spec
+dbSpecs env = describe "Databse SQL query correctness" $ do
+    describe "Repo" $ do
+        describe "upsertRepos" $ do
```

Comment:
I propose to achieve the same hierarchy by using `joinSpecs` function. The function is quite composable, so you can reduce nestedness by a lot:

```haskell
dbSpecs = joinSpecs "Databse SQL query correctness"
    [ repoSpecs
    , issueSpecs
    ]

issueSpecs = joinSpecs "Issue"
    [ upsertIssues
    , getIssues
    ]

upsertIssues = describe "upsertIssues" $ do
    ... actual tests go here
```